src/documents/tutorial-usingStringTemplates/UsingStringTemplates.md

```diff
@@ -0,0 +1,297 @@
+              funcName: "fluid.stringTemplate",
+              args: ["{arguments}.0", "{that}.options.terms"]
+          }
+      }
+  });
+
+  var simpler = gpii.sandbox.variables.simpler();
+  console.log("transformed options:\n" + JSON.stringify(simpler.options.transformed, null, 2));
+```
+
+That code produces output like the following:
+
+```
+  transformed options:
+  {
+    "one": "The term named 'one' is set to 'base one'.",
+    "two": "The term named 'two' is set to 'base two'."
+  }
+```
+
+What makes this all work is our `invokers` block:
+
+```
+  invokers: {
+      parseTemplates: {
+          funcName: "fluid.transform",
+          args: ["{that}.options.templates", "{that}.transformTemplate"]
+      },
+      transformTemplate: {
+          funcName: "fluid.stringTemplate",
+          args: ["{arguments}.0", "{that}.options.terms"]
+      }
+  }
+```
+
+The `parseTemplates` invoker calls `fluid.transform`, which runs a single function against every item in a map and returns an map containing the transformed results.  We have configured `fluid.transform` to use our `transformTemplate` invoker to process each value in our map.  The `transformTemplate` invoker calls `fluid.stringTemplate` with a list of arguments.  The first argument is expected to point to an array of templates, and the second argument is expected to point to a map of variable names and replacement values.
+
+In our case, the map of variable names and replacement values is already contained in `{that}.options.terms`.  The first argument is supplied by`fluid.transform`, which will pass the value of each array member to our invoker.  Since `fluid.transform` will call our invoker with a single argument, we can use `{arguments}.0` to refer to the information passed by `fluid.transform` in our list of arguments.
+
+The final piece is handled by an [`expander` block](../ExpansionOfComponentOptions.md).
+
+```
+  transformed: {
+      expander: {
+          func: "{that}.parseTemplates"
+      }
+  },
+```
+
+The `expander` block will be replaced with the value returned by our `parseTemplates` invoker, and we will then have sensible output that we can work with in our own functions and IoC references.
+
+### Overriding options from a child component
+
+In the next example, we will look at creating a child component that overrides some of our variables in its default configuration.  Here's the updated code:
+
+```javascript
+  "use strict";
+  var fluid = fluid || require("infusion");
+  var gpii  = fluid.registerNamespace("gpii");
+
+  fluid.registerNamespace("gpii.sandbox.variables.base");
+  fluid.defaults("gpii.sandbox.variables.base", {
+      gradeNames: ["autoInit", "fluid.eventedComponent"],
+      terms: {
+          one: "base one",
+          two: "base two"
+      },
+      templates: {
+          one: "The term named 'one' is set to '%one'.",
+          two: "The term named 'two' is set to '%two'."
+      },
+      listeners: {
+          onCreate: [
```

Comment:
Review comment from the previous round recommended namespacing this listener by using the key "onCreate.log" - https://github.com/fluid-project/infusion-docs/blob/master/src/documents/InfusionEventSystem.md#namespaced-listeners